.travis.yml

```diff
@@ -77,6 +77,11 @@ matrix:
       env: RAILS_VERSION='~> 5.2.0'
     - rvm: 2.3.8
       env: RAILS_VERSION='~> 5.2.0'
+    - rvm: 2.2.10
+      env: RAILS_VERSION='~> 5.2.0'
+      allow_failure: true
+    - rvm: 2.2.10
+      env: RAILS_VERSION='5-2-stable'
```

Comment:
Why do we want to try stable and `~> 5.2.0`?

Reply:
`5-2-stable` is `2.2.10` compatible, it contains an unreleased bugfix for Ruby 2.2, `~> 5.2.0` is included but will fail, eventually when patched the pending spec will fail this build and we can remove the `5-2-stable` the same as the other versions.

Reply:
Ok. Thanks!